frontend/src/lib/email/CreateProjectRequest.svelte

```diff
@@ -40,4 +40,5 @@
     </mj-table>
 <!--    todo should have a url here-->
     <mj-button href={createUrl}>{$t('project.create.submit')}</mj-button>
+    <mj-text>{$t('emails.link_expiration_warning')}</mj-text>
```

Comment:
I don't think this link expires. It doesn't have a token in it.

Reply:
Fixed in commit 4c0348a3.